### PR TITLE
created bookings controller and view

### PR DESCRIPTION
**Pull request by jfaMan**
Please legit check! I probably didn't do something right.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class BookingsController < ApplicationController
+  def index
+    @bookings = Bookings.all
```
Comment (lenanamiki) [inline app/controllers/bookings_controller.rb]: `@bookings = Booking.all`

**Review by lenanamiki — CHANGES_REQUESTED** [review]
Please check #index

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (jfaMan) [reply app/controllers/bookings_controller.rb]: Spotted! Thanks

**Review by lenanamiki — APPROVED** [review]
Good!

**Pull request merged**